src/System.ServiceModel.NetNamedPipe/src/System/IO/PipeException.cs

```diff
@@ -28,6 +28,7 @@ public PipeException(string message, Exception inner)
         {
         }
 
+        [Obsolete(DiagnosticId = "SYSLIB0051")]
         protected PipeException(SerializationInfo info, StreamingContext context)
```

Comment:
This is changing api surface which will have a doc effect too. Can you use a pragma to ignore instead or is this a hard requirement to compile?